packages/@aws-cdk/toolkit-lib/lib/api/notices/notices.ts

```diff
@@ -27,9 +41,9 @@ export interface NoticesProps {
   readonly output?: string;
 
   /**
-   * Options for the HTTP request
+   * Options for the HTTPS requests made by Notices
    */
-  readonly httpOptions?: SdkHttpOptions;
+  readonly httpsOptions?: NoticesHttpsOptions;
```

Comment:
If you call it `httpOptions` I'll assume it applies to all flavors of HTTP requests.

If you call it `httpsOptions` I'll wonder where the `httpOptions` are.

I see how this is strictly correct but is it helpful?

Reply:
Will revert back.